plugins/jdbc-adapter-framework/src/main/java/org/polypheny/db/adapter/jdbc/JdbcUtils.java

```diff
@@ -236,6 +171,11 @@ private PolyValue getPolyValue( int i ) throws SQLException {
                             return PolyFloat.ofNullable( (Number) o );
                         case Types.DECIMAL:
                             return PolyBigDecimal.ofNullable( (BigDecimal) o );
+                        case Types.JAVA_OBJECT:
+                            if ( o instanceof net.postgis.jdbc.PGgeometry pGgeometry ) {
+                                return PolyGeometry.ofNullable( pGgeometry.getValue() );
+                            }
+                            // fallback
```

Comment:
This looks dangerous... If some adds another case, this could lead to unexpected behavior. Why not add a throw clause? Btw: Why is this JAVA_OBJECT?

Reply:
Strangely enough this is not longer present but here not marked as outdated.